### PR TITLE
Refactor return till label

### LINKED ISSUE #608: 🐞 Refactor Return till: : label
- Environment:
- Client:

### ✔️ Expected behavior
It should not duplicate any symbols. Also, we could add `year` field to the label

### ✖️ Actual behavior
- It shows us duplicated colon :: when looking at the item card. 
- should make new labels acc to design

### Screenshot / Descriptions
We've just got new labels design
image
image

https://www.figma.com/file/pxj9Hhk5AdOxXWJP0dq4gT/Library-App-Spec?node-id=4993%3A15165&t=fcdHvTZesPT8UWxp-0

### PR DESCRIPTION
**Pull request by Chichmarin17**
Closes #608 
### How it looks now
image
image
image
image

### CODE REVIEW

File: packages/frontapp/src/app/hooks/useBookStatus.ts [hunk 1]
```diff
@@ -10,28 +10,34 @@ const useBookStatus = (
...
-    const day = `${getDates(created_at).returnDate.getDate()}`.padStart(2, '0');
-    const month = `${getDates(created_at).returnDate.getMonth() + 1}`.padStart(
-      2,
-      '0'
-    );
-    return `${t('Statuses.Busy')} : ${day}.${month}`;
-  } else if (currentStatus === StatusTypes.OVERDUE) {
-    return t('Statuses.Overdue');
+      return t('Statuses.Free');
+    case StatusTypes.BUSY || StatusTypes.PROLONG: {
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/hooks/useBookStatus.ts:22]: This will always result in `case 'Busy'`, Prolong will be ignored

**Review by IAlexHilimovich — CHANGES_REQUESTED** [review]

File: packages/frontapp/src/app/globalUI/Status/index.tsx [hunk 2]
```diff
@@ -8,9 +8,29 @@ interface IStyledBookStatusProps {
...
+        return colors.bg_error;
+      case StatusTypes.PENDING:
+        return colors.bg_own_claim;
+      default:
+        return '';
+    }
+  }};
+  border-radius: ${dimensions.xs_2};
   @media (max-width: ${dimensions.phone_width}) {
     width: fit-content;
```
Comment (IAlexHilimovich) [inline packages/frontapp/src/app/globalUI/Status/index.tsx:33]: Based on the design, `width: fit-content` should be for desktop resolution too

File: packages/frontapp/src/app/hooks/useBookStatus.ts (hunk 1 above)
Comment (Chichmarin17) [reply packages/frontapp/src/app/hooks/useBookStatus.ts]: nice shot, fixed

File: packages/frontapp/src/app/globalUI/Status/index.tsx (hunk 2 above)
Comment (Chichmarin17) [reply packages/frontapp/src/app/globalUI/Status/index.tsx]: moved style out of media query

**Review by IAlexHilimovich — APPROVED** [review]

**Pull request merged**